### PR TITLE
Linter cleanup

### PR DESCRIPTION
**Pull request by ShayNehmad**
# What does this PR do? 

Lower the number of linter warnings so that when someone adds new ones in their PR it's easier to locate and fix the new issues.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing? 

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Ran monkey just to see that the Python boots up - no logic changes
* [ ] If applicable, add screenshots or log transcripts of the feature working

## Explain Changes

Are the commit messages enough? If not, elaborate.

### CODE REVIEW

File: monkey/common/network/network_range.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def __iter__(self):
         """
         base_range = self.get_range()
         if self._shuffle:
-            random.shuffle(base_range)
+            random.shuffle(base_range)  # noqa: DUO102
```
Comment (VakarisZ) [inline monkey/common/network/network_range.py:31]: This about random being unsafe?

File: monkey/infection_monkey/utils/windows/hidden_files.py [hunk 2]
```diff
@@ -51,7 +51,7 @@ def get_winAPI_to_hide_files():
         fileCreation = win32file.CREATE_ALWAYS  # overwrite existing file
         fileFlags = win32file.FILE_ATTRIBUTE_HIDDEN  # make hidden
 
-        hiddenFile = win32file.CreateFile(HIDDEN_FILE_WINAPI,
+        _ = win32file.CreateFile(HIDDEN_FILE_WINAPI,
```
Comment (VakarisZ) [inline monkey/infection_monkey/utils/windows/hidden_files.py:54]: Why store return value at all if unused?

File: monkey/monkey_island/cc/encryptor.py [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import base64
 import os
 
-from Crypto import Random
-from Crypto.Cipher import AES
+# PyCrypto is deprecated, but we use pycryptodome, which uses the exact same imports but it maintained
+from Crypto import Random  # noqa: DOU133
+from Crypto.Cipher import AES  # noqa: DOU133
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/encryptor.py:6]: maybe except this rule altogether?

File: monkey/monkey_island/cc/main.py [hunk 4]
```diff
@@ -1,35 +1,38 @@
 import logging
 import os
-import os.path
 import sys
 import time
+from pathlib import Path
 from threading import Thread
 
-MINIMUM_MONGO_DB_VERSION_REQUIRED = "4.2.0"
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/main.py:8]: Ok, so where did this go?

**Review by VakarisZ — APPROVED** [review]
Looks good, but now I'll gone have to do a hell of a lot testing before next release :D

File: monkey/infection_monkey/utils/windows/hidden_files.py (hunk 2 above)
Comment (ShayNehmad) [reply monkey/infection_monkey/utils/windows/hidden_files.py:54]: True, fixed

File: monkey/monkey_island/cc/encryptor.py (hunk 3 above)
Comment (ShayNehmad) [reply monkey/monkey_island/cc/encryptor.py:6]: I'd rather not - new code should try to use different encryption algos and libraries if we can

**Pull request merged**